### ISSUE TITLE
Show creators in project preview

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#606]
### What are you trying to do that you can't?

When teachers are overseeing a gallery of projects, they need to be able to identify which of their students are working on specific projects. Project previews should show the owner and collaborator usernames, if they are available, in the preview.

### What is your idea?

The basic design would:

- [ ] Update `ProjectPreview` to render owner and collaborator usernames with a `CreatorView`

**Comment by awang040403** [comment]
Hi @amyjko, I talked with you this afternoon and am interested in working on this over the winter break. Could you please assign me to it?

**Comment by amyjko** [comment]
Yes, I can assign you @awang040403. This issue is high priority and needs to be in production by January 17th, so work should begin on it immediately. Comment here if that's not possible.

**Comment by awang040403** [comment]
Hi @amyjko, alright. I will let you know ahead of time I am having trouble finishing it on time. I have a quick question to begin the issue. wordplay-main\src\components\app\ProjectPreview.svelte is the file location that I assume to implement the feature, while would the project preview page be the screenshot or is it the gallery? 
![image](https://github.com/user-attachments/assets/6d3e16e7-989f-4f79-a0a2-12070e799d43)
https://wordplay.dev/galleries Or would it solely be presented in teacher's account?
Thank you for your clarification!

**Comment by amyjko** [comment]
Yes, `ProjectPreview` is where we want to show creators. We want to show them on all pages though, whenever a project has a creator. (It doesn't matter what page the preview appears on).

**Comment by awang040403** [comment]
Hello @amyjko, I have made some progress on editing ProjectPreview, while I am unable to test how my work looks because I do not have access to the firebase on my computer's environment. Therefore, I am unable to see the projects that other people created or save my own work when running wordplay with my own environment. Could you please give me an account or permission with the firebase to test my work? Thank you for your help!
![image](https://github.com/user-attachments/assets/10ec2acb-79db-44be-a659-a62ba4eb199e)

**Comment by amyjko** [comment]
You should be able to run the Firebase emulator following the instructions on the wiki:

https://github.com/wordplaydev/wordplay/wiki/Firebase

Let me know if you run into problems.

**Comment by awang040403** [comment]
Thank you Professor Ko! I will follow the instructions and let you know if I have further issues that I cannot solve.

**Comment by amyjko** [comment]
@awang040403 How are things going on this issue? I saw you closed the draft PR for this. Since this is a critical path issue for Adrienne's class, it'd be great to know the status.
